study.md

```diff
@@ -43,21 +42,19 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
-```
+View-state is like the DOM but for data on application interactions. It's a view of the dynamics of page loading. A view is an instance of viewstate. A router helps manage view-state by providing relevant views to the developer.```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
-```
+SPAs give the user what they expect from a web page. The user expects the page to stay "there" when they interact with it and input their info/ideas. If the user was to put a review on amazon of 3 stars and the page refreshed, that would be double plus uncool.
+Another advantage of SPAs is that it makes the coding more simple, or as devs prefer, "with more ease". It seperates the markup from the data and logic so it's easier to debug and revisit.```
```

Comment:
SPA's are also a much smoother experience for the user as your not always refreshing the page when you change the view state.